Unit tests failing in stats view

The tests currently running on Travis seem to be failing with the following::

```
FAIL: test_default_stats_get (test_stats_view.AdminStatsTestCase)
No start/end provided: Currently active users (3 yrs) returned
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/centos/streamwebs/streamwebs_frontend/streamwebs/tests/views/test_stats_view.py", line 201, in test_default_stats_get
    len(self.expected_user_stats['default']))
AssertionError: 2 != 3

======================================================================
FAIL: test_default_stats_post (test_stats_view.AdminStatsTestCase)
No start/end provided: Currently active users (3 yrs) returned
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/centos/streamwebs/streamwebs_frontend/streamwebs/tests/views/test_stats_view.py", line 174, in test_default_stats_post
    len(self.expected_user_stats['default']))
AssertionError: 2 != 3
```

I've tried running this test on an older version of the branch and it still fails so I suspect this might be related to some database migrations changing something.

--- a/streamwebs_frontend/streamwebs/fixtures/sample_users.json
+++ b/streamwebs_frontend/streamwebs/fixtures/sample_users.json
@@ -94,7 +94,7 @@
     "pk": 6,
     "fields": {
         "password": "pbkdf2_sha256$24000$ZSuSdEB5HP2B$nvgypbaHwqk00ZrsB55HWlnuIRdithfNuFCHw2ZgsEU=",
-        "last_login": "2014-10-31T13:45:00",
+        "last_login": "2017-10-31T13:45:00",
         "is_superuser": false,
         "username": "aithai",
         "first_name": "aileeen",

